core/src/main/java/org/mskcc/cbio/portal/util/ValueParser.java

```diff
@@ -598,9 +598,9 @@ public String getCopyNumberStyle() {
             case Diploid:
                return "no_change";
             case HemizygouslyDeleted:
-               return "hetloss_2";
```

Comment:
This is another file that hasn't seen changes since the early Cretaceous. I don't think we should change it.

Reply:
Reverted change to file